src/v3/preact.config.js

```diff
@@ -128,76 +121,60 @@ export default {
       'index',
     );
 
-    // TODO: integrate with rollup bundling system once migrate to the widget repo
-    // preact-cli still stays with webpack v4, which seems to have issue to understand `exports` field correctly in auth-js' subdeps
-    // using ESM bundle of auth-js is required to enable tree-shaking
+    // TODO: integrate with rollup bundling system once migrate to the widget
+    // repo preact-cli still stays with webpack v4, which seems to have issue to
+    // understand `exports` field correctly in auth-js' subdeps using ESM bundle
+    // of auth-js is required to enable tree-shaking
     // https://github.com/preactjs/preact-cli/issues/1579
-    config.resolve.alias['@okta/okta-auth-js'] = resolve(
-      '..',
-      '..',
+    config.resolve.alias['@okta/okta-auth-js'] = rootResolve(
       'node_modules',
       '@okta',
       'okta-auth-js',
       'esm',
       'esm.browser.js',
     );
 
-    // broadcast-channel esnode bundle is loaded by default, browser one should be used
-    config.resolve.alias['broadcast-channel'] = resolve(
-      '..',
-      '..',
+    // broadcast-channel esnode bundle is loaded by default, browser one should
+    // be used
+    config.resolve.alias['broadcast-channel'] = rootResolve(
       'node_modules',
       'broadcast-channel',
       'dist',
       'esbrowser',
       'index.js',
     );
 
-    if (env.production) {
-      config.output.libraryTarget = 'umd';
-    }
-
     // polyfills for jsonforms
-    config.plugins.push(
-      new DefinePlugin({
-        // https://github.com/APIDevTools/json-schema-ref-parser#browser-support
-        process: {
-          browser: true,
-        },
-        Buffer: buffer,
-      }),
-    );
-
-    config.devServer = {
-      host: HOST,
-      static: [PLAYGROUND, TARGET, {
-        staticOptions: {
-          watchContentBase: true
-        }
-      }],
-      historyApiFallback: true,
-      // FIXME CSP prevents scripts from loading in dev mode
-      // headers: { 'Content-Security-Policy': `script-src http://${HOST}:${DEV_SERVER_PORT}` },
-      compress: true,
-      port: DEV_SERVER_PORT,
-      proxy: [{
-        context: [
-          '/oauth2/',
-          '/api/v1/',
-          '/idp/idx/',
-          '/login/getimage',
-          '/sso/idps/',
-          '/app/UserHome',
-          '/oauth2/v1/authorize',
-          '/auth/services/',
-          '/.well-known/webfinger'
-        ],
-        target: `http://${HOST}:${MOCK_SERVER_PORT}`
-      }],
-    },
+    // https://github.com/APIDevTools/json-schema-ref-parser#browser-support
+    config.plugins.push(new DefinePlugin({ process: { browser: true }, Buffer }));
```

Comment:
we'll be removing this in the remove jsonforms branch so you can avoid updating this if you want

Reply:
Saw that. I'll deal with it at merge time